docker-compose.yml

```diff
@@ -130,6 +130,27 @@ services:
       INFLUXDB_INIT_PWD: "${TTN_DASHBOARD_INFLUXDB_ADMIN_PASSWORD:-!notset}"
       PRE_CREATE_DB: "${TTN_DASHBOARD_INFLUXDB_INITIAL_DATABASE_NAME:-demo}"
 
+  ############# Murugan Changes for influxdb backup
```

Comment:
do not put "myname changes for influx db back" comments. This info is held by version control. If you have it while developing, OK, but please clean up before committing.